### DISCUSSION TITLE
Differences between SliverList vs ListView in Flutter

### DISCUSSION THREAD

**Discussion by diwasche** [community/community#29954] [Mobile] [answered]
[Differences between SliverList vs ListView in Flutter](https://stackoverflow.com/questions/50367640/differences-between-sliverlist-vs-listview-in-flutter)

**Comment by surajadkhari** [comment, marked as answer]
There's almost no difference.

ListView is a SliverList. Same with GridView, which is a SliverGrid.

They are doing exactly the same thing. The only difference between them is that SliverList is a sliver, not a widget. Which means it's used inside a ScrollView, usually CustomScrollView.

ListView is nothing else but a biding of SliverList to transform it into a Widget to make it usable alongside other widgets such as Row/Container.

**Reply by lapislazuli21** [reply]
Maybe don't copy someone's answer word for word.

https://stackoverflow.com/questions/50367640/differences-between-sliverlist-vs-listview-in-flutter

**Comment by rahulshahDEV** [comment]
There's a Difference between silver list and ListView when it comes to performance of app.

Example :  [listview vs silverlist](https://github.com/rahulshahDEV/list-view-vs-silver-view/tree/master/silver)

silverlist support lazy loading that helps to load only smaller chunk of data visible in screen where listview loads all data that may affect in performance of app

Youtube : [video](https://www.youtube.com/watch?v=LUqDNnv_dh0)
